starter-code/functions-and-arrays.js

```diff
@@ -104,9 +145,21 @@ console.log(hasMatter);
 var hasDog = doesWordExist(words, "dog");
 console.log(hasDog);
 
+
+
 // Counting Repetion
 function howManyTimes (words, word) {
```

Comment:
This will work fine but it will modify the array `words` in place. By doing `wordsBis = words` you are not copying the array; you are just giving it a new name.

A simpler approach would simply to loop on the array and count the number of times the current element is equal to `word`.